### DISCUSSION TITLE
MEETING: ecatalogue Working Group - 16 June 2022

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#600] [MEETINGS]
# Agenda: Tissue/Genetics Tab
## Review Workflow and tabs involved
What values to autofill for Template?
What values to include as manual input on Template?

**Comment by fmjjones** [comment]
# Meeting Notes
**attendees: AF, CM, SM, JM, RSB, KGJ, MT, JJ, BM**
-Review of workflow for creating Tissue Samples as child records.
- Review of Prep Tab
##Questions
- Where would Fishes record the number assigned to a specimen that has a tissue sample? The number is different from the tissue number and is assigned as a suffix to the catalog #. 

# Next/Homework:

**Review Workflow and tabs involved**
- What values to autofill for Template?
- What values to include as manual input on Template?
**Review Tabs to include on Tissue Tab switching**
- - Preliminary:

- - - AllCatGTab
- - - AllDesKTab
- - - AllRelTab
- - - AllRelNhTab
- - - AllTisTab
- - - AllGenTab
- - - AllPreMTab
- - - AllDeaTab
- - - AllPheTab
- - - AllPrv2Tab

**What information is hiding in remarks that should be addressed now while designing tabs and workflows?**
**Genetics Tab discussion on changes**
- On Genetics tab: Consider suggested changes; What’s missing or needed?
